### PR TITLE
Update Grunt configuration.

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
 - Updates Grunt configuration (alongside changes in DoSomething/dosomething#3751).

For review: @DoSomething/front-end

### CODE REVIEW

File: tasks/aliases.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "default": ["build", "lint", "test", "watch"],
```
Comment (weerd) [inline tasks/aliases.json]: Do you need to specify a `watch` task?

**Comment by weerd** [comment]
Overall :+1:

File: tasks/sass.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  function writeFile(path, contents) {
+    if(grunt.file.exists(path)) {
+      grunt.file.delete(path);
+    }
+
+    grunt.file.write(path, contents);
+  }
+
+  grunt.registerTask('sass', function() {
+    console.log(sass.info());
```
Comment (sbsmith86) [inline tasks/sass.js]: should this console.log be removed?

File: tasks/aliases.json (hunk 1 above)
Comment (DFurnes) [inline tasks/aliases.json]: Nope, points to the `watch` multitask in [watch.js](https://github.com/DFurnes/neue/blob/efe41fb0bcb4dfd10f0718c187fa20120e107e6a/tasks/watch.js).

Comment (weerd) [inline tasks/aliases.json]: Hokay!

File: tasks/sass.js (hunk 2 above)
Comment (DFurnes) [inline tasks/sass.js]: Replaced with a `grunt.log.writeln`. :+1:

**Pull request merged**